docs/plugins.adoc

```diff
@@ -13,6 +13,16 @@ include::../plugins.yaml[tag="external_plugins"]
 organization. Be aware however, that adding an external plugin both to the repository and its organization link:https://github.com/kubernetes/test-infra/blob/7de525b1f6943e5d08d9a127b0b668cec404c665/prow/plugins/plugins_test.go#L143[will result in an error].
 <2> You can limit events dispatched by hook to your plugin.
 
+==== Creating custom plugin hint messages
+
+To enable customized plugin hint comment messages edit plugin's configuration file (eg. test-keeper.yaml) adding the `plugin_hint` property.
```

Comment:
We don't really say what they are for and why they are useful, only what to do to add them. Not sure if someone would find this information handy in this shape.